app/services/invoices/compute_amounts_from_fees.rb

```diff
@@ -29,7 +29,7 @@ def call
         invoice.fees_amount_cents - invoice.progressive_billing_credit_amount_cents - invoice.coupons_amount_cents
       )
 
-      taxes_result = if customer_provider_taxation?
+      taxes_result = if customer_provider_taxation? && invoice.fees.any?
```

Comment:
is the fall through here logical? e.g. we'd call `Invoices::ApplyTaxesService` when there are no fees?

Reply:
I see the point but the goal of this PR was just to skip fetching taxes for `invoice.fees = []` case. In that case, we want to use the flow where all other invoices end-up and where Lago taxes are used (to avoid Anrok to raise validation error for invoices without fees).
This PR ws aiming to fix only this case and not to change any logic for invoices without fees in general.

Reply:
Okay, just double checking, I think that is good enough 👍